### PR TITLE
Kg/pg reorg

### PR DESCRIPTION
**Pull request by kagarmoe**
### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/external_postgres_upgrade_steps.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+[menu]
+  [menu.automate]
+    title = "Upgrade External PostgreSQL"
+    parent = "automate/reference"
+    identifier = "automate/reference/external_postgres_upgrade_steps.md.md External PostgreSQL Upgrade Steps"
++++
+
+This guide covers upgrading services used by Chef Automate.
+
+## Upgrade PostgreSQL for Amazon RDS
```
Comment (tas50) [inline components/docs-chef-io/content/automate/external_postgres_upgrade_steps.md:16]: This is backwards. It's 'Amazon RDS for PostgreSQL'

File: components/docs-chef-io/content/automate/external_postgres_upgrade_steps.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+### Migration Planning
+
+The upgrade process for PostgreSQL from 9.6 to 13.3 requires a one-time downtime to vacuum, upgrade, and re-index the database. The entire upgrade operation takes about five minutes for each 2GB of data in PostgreSQL. This process may take longer depending on your server hardware and the size of the node objects in Chef Automate.
+
+### Requirements
+
+This upgrade guide was written for systems with:
+
+- Single PostgreSQL 9.6 installation
+- Running on virtual machine such as EC2 or on a single physical machine
```
Comment (tas50) [inline components/docs-chef-io/content/automate/external_postgres_upgrade_steps.md:35]:
```suggestion
- Running on virtual machine such as an EC2 instance or on a single physical machine
```

EC2 is not a VM

File: components/docs-chef-io/content/automate/external_postgres_upgrade_steps.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
++++
+title = "External PostgreSQL Upgrade"
```
Comment (tas50) [inline components/docs-chef-io/content/automate/external_postgres_upgrade_steps.md:2]: Parts of this document reference PostgreSQL 13.3 and parts reference 13.4. Is that intentional?

Comment (kagarmoe) [reply components/docs-chef-io/content/automate/external_postgres_upgrade_steps.md]: We're using 13.4 in this doc.

**Review by vivekshankar1 — APPROVED** [review]
thanks @kagarmoe , this looks good

File: components/docs-chef-io/content/automate/monitoring.md [hunk 4]
```diff
@@ -17,11 +17,11 @@ Use the authenticated https endpoint `/status` to monitor your Chef Automate ins
 
 ## Checking the Status Endpoint
 
-The authenticated endpoint `/status` provides status for the overall Chef Automate installation as well as for its component services.
+The authenticated endpoint `/status` provides status for the Chef Automate installation as well as for its component services.
 When all Chef Automate component services are up, `/status` returns a response code of 200.
 Otherwise, `/status` returns 500.
 
-The status of a service can be `OK`, `UNKNOWN`, or `CRITICAL`, and is reflected in the JSON output:
+The status of a service can be `OK`, `UNKNOWN`, or `CRITICAL`, and is show in the JSON output:
```
Comment (vivekshankar1) [inline components/docs-chef-io/content/automate/monitoring.md:24]:
```suggestion
The status of a service can be `OK`, `UNKNOWN`, or `CRITICAL`, and is shown in the JSON output:
```

**Review by vivek-yadav — APPROVED** [review]

**Pull request merged**